gitpod-io/gitpod
# Migrate dashboards from jsonnet to json

Signed-off-by: ArthurSens <arthursens2005@gmail.com>

## Description

Almost all our dashboards are written in JSON today, this PR migrates the left overs from Jsonnet to JSON to make it easier to maintain and develop

## Related Issue(s)

Fixes #

## How to test

* Open a workspace from this PR
* Access grafana via `kubectl port-forward -n monitoring-satellite svc/grafana 3000`
* Verify dashboard in the "Cross-team" folder are still there and functional

## Release Notes

```release-note
NONE
```

## Documentation


## Werft options:

      If enabled this will build `install/preview`
      Valid options are `all`, `workspace`, `webapp`, `ide`
